test/keyword_type_test.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const chai = require('chai');
+chai.use(require('chai-http'));
+
+const server = require('../app');
+const auth   = require('../app/lib/authentication');
+const knex    = require('../app/lib/bookshelf').knex;
+
+const testdata = require('../seeds/test_data.json');
+
+
+describe('KeywordType Controller', function() {
+
+    before('Setup SQLite memory database', function() {
+        return knex.migrate.latest();
+    });
+
+    beforeEach('Populate SQLite memory DB with fresh test data', function() {
+        return knex.seed.run();
+    });
+    describe('GET Keyword Type', function(){
+        it('Successfully retrieves KeywordType by id in the database', function(done) {
+            let testdata=testdata.keyword_types[0];
```

Comment:
Two things:
 1. You're redefining testdata here, so this is going to cause unexpected behavior.
 2. You are right to use testdata.keyword_types, but we're testing to make sure we get _all_ of them, not just the one at index 0.